hack/cherry_pick_pull.sh

```diff
@@ -124,17 +133,9 @@ done
 gitamcleanup=false
 
 function make-a-pr() {
-  echo "+++ Now you must propose ${NEWBRANCH} as a pull against ${BRANCH} (<--- NOT MASTER)."
-  echo "    You are constructing a pull against the upstream release branch! To do"
-  echo "    this in the GitHub UI, when you get to the 'Comparing Changes' screen, keep the"
-  echo "    'base fork' as GoogleCloudPlatform/kubernetes, but change the 'base' to e.g. 'release-1.0',"
-  echo "    presumably ${BRANCH}. (This selection is near the top left.)"
-  echo
-  echo "    Use this subject: 'Automated cherry pick of ${PULLSUBJ}' and include a justification."
-  echo
-  echo "    Note: the tools actually scrape the branch name you just pushed, so don't worry about"
-  echo "    the subject too much, but DO keep at least ${NEWBRANCHREQ} in the remote branch name."
-  echo
+  echo "+++ Creating a pull request on github"
+  echo hub pull-request -m "Automated cherry pick of ${PULLSUBJ}" -h ${GITHUB_USER}:${NEWBRANCH} -b GoogleCloudPlatform:`basename ${BRANCH}`
+  hub pull-request -m "Automated cherry pick of ${PULLSUBJ}" -h ${GITHUB_USER}:${NEWBRANCH} -b GoogleCloudPlatform:`basename ${BRANCH}`
```

Comment:
Do we have somewhere to dump a link to https://google-styleguide.googlecode.com/svn/trunk/shell.xml ? Maybe should just have a style guide landing page for k8s.

(In the case of bash, we've settled on basically that style guide with the exception that we strongly prefer `.sh` over naked script names.)